notifications.bs

```diff
@@ -189,6 +209,10 @@ or vibration pattern that is not otherwise accessible to the end user.
   <li><p>If <var>options</var>'s <code>sticky</code> is true, set
   <var>notification</var>'s <a>sticky preference flag</a>.
 
+  <li><p>Set <var>notification</var>'s list of <a>actions</a> to the first
+  {{Notification/maxActions}} entries from <var>options</var>'s
```

Comment:
maxActions will have to be some concept if we want to use it like this. Referencing code directly doesn't work well since it can be overridden by script.